modules/otel-config-instrumentation.adoc

```diff
@@ -169,6 +214,14 @@ instrumentation.opentelemetry.io/inject-dotnet: "true"
 [id="otel-configuration-of-go-auto-instrumentation_{context}"]
 == Configuration of the Go auto-instrumentation
 
+:FeatureName: The Go auto-instrumentation
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+By default, this feature injects unsupported, upstream instrumentation libraries.
```

Comment:
```suggestion
By default, this feature injects unsupported upstream instrumentation libraries.
```

I don't think comma is required here. Hence I suggest to remove that.

Reply:
Thank you for the suggestion. Using a comma here is correct in my understanding of punctuation.